Riot/Modules/Room/RoomViewController.m

```diff
@@ -3047,174 +3039,6 @@ - (RoomTimelineCellIdentifier)cellIdentifierForCellData:(MXKCellData*)cellData a
-            }
-        }
-    }
-    else if (roomBubbleCellData.getFirstBubbleComponentWithDisplay.event.isEmote)
-    {
-        if (bubbleData.isIncoming)
-        {
-            if (bubbleData.isPaginationFirstBubble)
-            {
-                if (bubbleData.shouldHideSenderName)
-                {
-                    cellIdentifier = showEncryptionBadge ? RoomTimelineCellIdentifierIncomingEmoteEncryptedWithPaginationTitleWithoutSenderName : RoomTimelineCellIdentifierIncomingEmoteWithPaginationTitleWithoutSenderName;
-                }
-                else
-                {
-                    cellIdentifier = showEncryptionBadge ? RoomTimelineCellIdentifierIncomingEmoteEncryptedWithPaginationTitle : RoomTimelineCellIdentifierIncomingEmoteWithPaginationTitle;
-                }
-            }
-            else if (bubbleData.shouldHideSenderInformation)
-            {
-                cellIdentifier = showEncryptionBadge ? RoomTimelineCellIdentifierIncomingEmoteEncryptedWithoutSenderInfo : RoomTimelineCellIdentifierIncomingEmoteWithoutSenderInfo;
-            }
-            else if (bubbleData.shouldHideSenderName)
-            {
-                cellIdentifier = showEncryptionBadge ? RoomTimelineCellIdentifierIncomingEmoteEncryptedWithoutSenderName : RoomTimelineCellIdentifierIncomingEmoteWithoutSenderName;
-            }
-            else
-            {
-                cellIdentifier = showEncryptionBadge ? RoomTimelineCellIdentifierIncomingEmoteEncrypted : RoomTimelineCellIdentifierIncomingEmote;
-            }
-        }
-        else
-        {
-            if (bubbleData.isPaginationFirstBubble)
-            {
-                if (bubbleData.shouldHideSenderName)
-                {
-                    cellIdentifier = showEncryptionBadge ? RoomTimelineCellIdentifierOutgoingEmoteEncryptedWithPaginationTitleWithoutSenderName : RoomTimelineCellIdentifierOutgoingEmoteWithPaginationTitleWithoutSenderName;
-                }
-                else
-                {
-                    cellIdentifier = showEncryptionBadge ? RoomTimelineCellIdentifierOutgoingEmoteEncryptedWithPaginationTitle : RoomTimelineCellIdentifierOutgoingEmoteWithPaginationTitle;
-                }
-            }
-            else if (bubbleData.shouldHideSenderInformation)
-            {
-                cellIdentifier = showEncryptionBadge ? RoomTimelineCellIdentifierOutgoingEmoteEncryptedWithoutSenderInfo : RoomTimelineCellIdentifierOutgoingEmoteWithoutSenderInfo;
-            }
-            else if (bubbleData.shouldHideSenderName)
-            {
-                cellIdentifier = showEncryptionBadge ? RoomTimelineCellIdentifierOutgoingEmoteEncryptedWithoutSenderName : RoomTimelineCellIdentifierOutgoingEmoteWithoutSenderName;
-            }
-            else
-            {
-                cellIdentifier = showEncryptionBadge ? RoomTimelineCellIdentifierOutgoingEmoteEncrypted : RoomTimelineCellIdentifierOutgoingEmote;
-            }
-        }
-    }
```

Comment:
If you can, please add this code or comment it

Reply:
The code is still here (and commented) but in another place in the file (from line 2876 to 3041).
It's a presentation issue from GitHub.